### ISSUE TITLE
Error starting tiflash on Ubuntu 22.04.2 LTS (Jammy Jellyfish) due to GLIBC version mismatch

### ISSUE THREAD

**Issue by dulao5** [pingcap/tiflash#6853]
## Bug Report

I am trying to start tiflash on Ubuntu 22.04.2 LTS (Jammy Jellyfish), but I am getting the following error:
```
/root/.tiup/components/tiflash/v6.5.0/tiflash/tiflash: /lib/aarch64-linux-gnu/libpthread.so.0: version `GLIBC_PRIVATE' not found (required by /root/.tiup/components/tiflash/v6.5.0/tiflash/tiflash)


cat /etc/os-release
PRETTY_NAME="Ubuntu 22.04.2 LTS"
NAME="Ubuntu"
VERSION_ID="22.04"
VERSION="22.04.2 LTS (Jammy Jellyfish)"
VERSION_CODENAME=jammy
ID=ubuntu
ID_LIKE=debian
...
```

### 1. Minimal reproduce step (Required)

```
git clone git@github.com:ti-click/docker-tidb-playground.git
cd docker-tidb-playground.git
```

```
vi .env

- TIDB_VERSION=v6.1.1
+ TIDB_VERSION=v6.5.0
```

```
vi docker-compose.yml
```

```
diff --git a/docker-compose.yml b/docker-compose.yml
index 756f36a..f674bcb 100644
--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -4,11 +4,11 @@ services:
     environment:
       TZ: "Asia/Tokyo"
     image: ubuntu:latest
-    platform: linux/amd64
+    #platform: linux/amd64
     build:
       context: .
       dockerfile: Dockerfile.tidb
-    command: /bin/sh -c "exec /root/.tiup/bin/tiup playground ${TIDB_VERSION} --host 0.0.0.0 --tag=devenv"
+    command: /bin/sh -c "exec /root/.tiup/bin/tiup playground ${TIDB_VERSION} --host 0.0.0.0 --tag=devenv --db 1 --kv 1 --pd 1 --tiflash 1"
     ports:
```

```
docker compose up --build -d

docker compose logs tidb
```

### 2. What did you expect to see? (Required)
I expected the tiflash-server process to start successfully.

### 3. What did you see instead (Required)

I encountered the following error:

> /root/.tiup/components/tiflash/v6.5.0/tiflash/tiflash: /lib/aarch64-linux-gnu/libpthread.so.0: version `GLIBC_PRIVATE' not found (required by /root/.tiup/components/tiflash/v6.5.0/tiflash/tiflash)

This error is preventing me from starting tiflash, and I am not sure how to resolve it. Any help or suggestions would be greatly appreciated. 
Thank you.

### 4. What is your TiFlash version? (Required)

v6.5.0

**Comment by Lloyd-Pottiger** [comment]
Have fixed by https://github.com/pingcap/tiflash/pull/6572, but not merged in release-6.5. Please use nightly instead.

**Comment by JaySon-Huang** [comment]
openEuler 22.03 LTS SP1 also suffer from this issue. And we verified it is fixed since v6.6.0
```
# cat /etc/system-release
openEuler release 22.03 (LTS-SP1)

# /root/.tiup/bin/tiup tiflash:v6.5.4 version
Starting component `tiflash`: /root/.tiup/components/tiflash/v6.5.4/tiflash/tiflash version
/root/.tiup/components/tiflash/v6.5.4/tiflash/tiflash: /usr/lib64/libpthread.so.0: version `GLIBC_PRIVATE' not found (required by /root/.tiup/components/tiflash/v6.5.4/tiflash/tiflash)

# /root/.tiup/bin/tiup tiflash:v6.6.0 version
Starting component `tiflash`: /root/.tiup/components/tiflash/v6.6.0/tiflash/tiflash version
TiFlash
Release Version: v6.6.0
Edition:         Community
Git Commit Hash: 77c4d0f7ca15b6d77e0d82b9ad5cbf0d7925cc91
Git Branch:      heads/refs/tags/v6.6.0
UTC Build Time:  2023-02-09 16:32:20
Enable Features: jemalloc sm4(GmSSL) asimd unwind thinlto
Profile:         RELWITHDEBINFO

Raft Proxy
Git Commit Hash:   6444536dc45ccf50949ac47f10ced164578ddf55
Git Commit Branch: HEAD
UTC Build Time:    2023-02-09 16:38:52
Rust Version:      rustc 1.67.0-nightly (96ddd32c4 2022-11-14)
Storage Engine:    tiflash
Prometheus Prefix: tiflash_proxy_
Profile:           release
```
